### PR TITLE
reset win capture on winrt closed event

### PR DESCRIPTION
**Pull request by summeroff**

### CODE REVIEW

**Comment by EddyGharbi** [comment]
Could you rebase to the `streamlabs` branch?

**Comment by summeroff** [comment]
changed to streamlabs branch

File: slobs_CI/install-script-win.cmd [hunk 1]
```diff
@@ -1,4 +1,4 @@
-set DEPS=dependencies2017.2
+set DEPS=dependencies2017.3
```
Comment (EddyGharbi) [inline slobs_CI/install-script-win.cmd:1]: What did we update in our dependencies?

Comment (summeroff) [reply slobs_CI/install-script-win.cmd:1]: It comes from https://github.com/stream-labs/obs-studio/pull/206 

My branch include this change as it initially started from streamlabs-25.0.8 branch.

Comment (EddyGharbi) [reply slobs_CI/install-script-win.cmd:1]: Sounds good, seems like this change has never been merged into the base branch which is the current preview.

**Review by EddyGharbi — APPROVED** [review]

**Pull request merged**